### PR TITLE
CLOUD-820 README update to mention documentation

### PR DESCRIPTION
**Pull request by nastena1606**
[CLOUD-820](https://jira.percona.com/browse/CLOUD-820) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

modified:   README.md

**CHANGE DESCRIPTION**
---
**Problem:** 
READMEs across all Operators are misaligned, having different structure, content and lacking links to Documentation

*Short explanation of the problem.* 
Update README to mention documentation

**Cause:**
*Short explanation of the root cause of the issue if applicable.*

**Solution:**
Added Docs links, updated READMe structure and content

*Short explanation of the solution we are providing with this PR.*

**CHECKLIST**
---
**Jira**
- [x] Is the Jira ticket created and referenced properly?
- [x] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [ ] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [ ] Is an E2E test/test case added for the new feature/change?
- [ ] Are unit tests added where appropriate?

**Config/Logging/Testability**
- [ ] Are all needed new/changed options added to default YAML files?
- [ ] Are the manifests (crd/bundle) regenerated if needed?
- [ ] Did we add proper logging messages for operator actions?
- [ ] Did we ensure compatibility with the previous version or cluster upgrade process?
- [ ] Does the change support oldest and newest supported PG version?
- [ ] Does the change support oldest and newest supported Kubernetes version?

[CLOUD-820]: https://perconadev.atlassian.net/browse/CLOUD-820?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,37 +23,49 @@ Whether you need to get a simple PostgreSQL cluster up and running, need to depl
 - Scheduled and manual backups
 - Integrated monitoring with [Percona Monitoring and Management](https://www.percona.com/software/database-tools/percona-monitoring-and-management)
 
-# Architecture
+Percona Operator is the command line tool. If you feel more comfortable with operating the Operator and database clusters via the web interface, there is [Percona Everest](https://docs.percona.com/everest/index.html) web-based database provisioning tool available for you. It automates day-to-day database management operations for you, reducing the overall administrative overhead. [Get started with Percona Everest](https://docs.percona.com/everest/quickstart-guide/quick-install.html)
```
Comment (spron-in) [inline README.md:26]:
```suggestion
Percona Operator is the command line tool. If you feel more comfortable with operating the Operator and database clusters via the web interface, there is [Percona Everest](https://docs.percona.com/everest/index.html) open-source web-based database provisioning tool available for you. It automates day-to-day database management operations for you, reducing the overall administrative overhead. [Get started with Percona Everest](https://docs.percona.com/everest/quickstart-guide/quick-install.html)
```

**Review by spron-in — COMMENTED** [review]
love it!

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| custom-extensions | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-640/7a31c2cb0/custom-extensions.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-640/7a31c2cb0/demand-backup.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-640/7a31c2cb0/init-deploy.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-640/7a31c2cb0/monitoring.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-640/7a31c2cb0/operator-self-healing.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-640/7a31c2cb0/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-640/7a31c2cb0/scheduled-backup.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-640/7a31c2cb0/self-healing.log) |
| start-from-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-640/7a31c2cb0/start-from-backup.log) |
| telemetry-transfer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-640/7a31c2cb0/telemetry-transfer.log) |
| upgrade-minor | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-640/7a31c2cb0/upgrade-minor.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-640/7a31c2cb0/users.log) |
| We run 12 out of 12|

commit: https://github.com/percona/percona-postgresql-operator/pull/640/commits/7a31c2cb0ad98c013019f4db13f7bb78aa91ad90
image: `perconalab/percona-postgresql-operator:PR-640-7a31c2cb0`

**Review by fiowro — DISMISSED** [review]
lgtm :)

**Review by hors — APPROVED** [review]

**Pull request merged**